### ISSUE TITLE
Error message displayed by flycheck-next-error quickly disappears from minibuffer

### ISSUE THREAD

**Issue by adamcohen2** [flycheck/flycheck#2080]
## Checklist

- [x] I have searched existing issues and StackOverflow for solutions.
- [x] I have read the [Flycheck manual][].
- [x] I have read the [Troubleshooting Guide][].
- [x] I have checked my issue has not been reported yet.

## Problem description

When I run `flycheck-next-error`, the error message displays very quickly in the minibuffer, and then disappears (see `Screenshots/Screen Recordings` section for example)

## Steps to reproduce

Execute `flycheck-next-error`

- [x] I have read https://emacs.stackexchange.com/questions/28429/how-do-i-troubleshoot-emacs-problems

## Expected behavior

The error message in the minibuffer should not disappear.

## Screenshots/Screen Recordings

https://github.com/flycheck/flycheck/assets/91114014/f35fb19e-c9fe-4e6d-b4b7-2a609346b585

## System configuration

<details><summary>Click to expand</summary>

```shell
Syntax checkers for buffer my_class_spec.rb in ruby-mode:

First checker to run:

  ruby-rubocop
    - may enable:         yes
    - executable:         Found at /Users/adam/.local/share/mise/shims/rubocop
    - configuration file: Found at "/Users/adam/Documents/programming/gitlab/gdk/gitlab/.rubocop.yml"
    - next checkers:      ruby-reek, ruby-rubylint

Checkers that could run if selected:

  ruby  select
    - may enable:    yes
    - executable:    Found at /Users/adam/.local/share/mise/shims/ruby
    - next checkers: ruby-rubylint

Checkers that are compatible with this mode, but will not run until properly configured:

  chef-foodcritic (automatically disabled) reset
    - may enable: no
    - may run:    nil
    - executable: Not found

  ruby-standard (automatically disabled) reset
    - may enable:         no
    - executable:         Not found
    - configuration file: Not found
    - next checkers:      ruby-reek, ruby-rubylint

  ruby-reek (automatically disabled) reset
    - may enable:         no
    - executable:         Not found
    - configuration file: Not found
    - next checkers:      ruby-rubylint

  ruby-rubylint (automatically disabled) reset
    - may enable:         no
    - executable:         Not found
    - configuration file: Not found

  ruby-jruby (automatically disabled) reset
    - may enable:    no
    - executable:    Not found
    - next checkers: ruby-rubylint

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled checkers.

--------------------

Flycheck version: 33snapshot (package: 20230306.414)
Emacs version:    29.0.92
System:           aarch64-apple-darwin22.5.0
Window system:    ns
```

</details>

Emacs configuration:
- [x] Plain Emacs / Custom configuration
- [ ] Spacemacs
- [ ] Doom Emacs
- [ ] Prelude
- [ ] Other shared configuration (please specify)

## Additional notes

[Flycheck manual]:  https://www.flycheck.org/en/latest/user/troubleshooting.html
[Troubleshooting guide]:  https://www.flycheck.org/en/latest/user/troubleshooting.html

**Comment by adamcohen2** [comment]
> The content of the minibuffer is being replaced by a suggested key binding. This can be remedied with:
> 
> (setq suggest-key-bindings nil)

Thank you!! This has been driving me crazy for years!! 🥳
